### DISCUSSION TITLE
Ask your questions here!

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2277] [Evennia Questions] [answered]
### Ask the community for Evennia help here! 

- Use the search-bar first - maybe your question was already addressed or you could jump in on an existing thread!
- Outright bugs-to-fix should be reported as an [bug report](https://github.com/evennia/evennia/issues/new/choose) instead of here.
- Feature-reqests should be, well, opened as a [feature request](https://github.com/evennia/evennia/issues/new/choose). If you specifically want to discuss a feature or 'probe the waters' for a future feature request, use the _Evennia Engine Development_ category.

[Click to start a new question thread](https://github.com/evennia/evennia/discussions/new) (select `Evennia questions` as category).

When you have reached enlightenment, don't forget to mark your thread as _answered_ by selecting the best answering post.

### Other help resources

- [Evennia documentation and tutorials](https://www.evennia.com/docs/latest/)
- You can also ask in the Evennia support chat channels on [discord](https://discord.gg/NecFePw) - you may get a faster response there, but if you can't hang around or want more of a 'persistent' discussion, this is the better place.

**Comment by kovitikus** [comment]
Shouldn't the categories have proper title capitalization? Such as `Evennia Questions` for example?

**Reply by Griatch** [reply]
Not sure that _is_ a proper capitalization :) But sure, I we can go with that.

**Comment by Griatch** [comment, marked as answer]
Testing out locking and answering. Edit - so one can answer one's own question, interesting.
